net/v2ray-core/Makefile

```diff
@@ -30,9 +30,10 @@ include $(INCLUDE_DIR)/package.mk
 include ../../lang/golang/golang-package.mk
 
 define Package/v2ray/template
-  TITLE:=A platform for building proxies to bypass network restrictions
+  TITLE:=Proxy platform to bypass censorship
```

Comment:
```suggestion
  TITLE:=A proxy platform to bypass network restrictions
```

Reply:
I guess you forget to update it. And please remember to update commit message too.